### PR TITLE
Fix checkout page auto scroll issue

### PR DESCRIPTION
**Pull request by Mayisha**
#### Changes proposed in this Pull Request
The changes made in the platform checkout auto redirection PR https://github.com/Automattic/woocommerce-payments/pull/4328 introduced a scroll issues. The checkout page scrolls down a little when the page is loaded. This PR fixes that and prevents the unwanted auto scroll.

#### Steps to create the issue
- Add some products to the cart and go to the checkout page.
- The checkout page automatically scrolls down a bit. 

#### Testing instructions
- Add some products to the cart and go to the checkout page.
- The checkout page should not scroll down

### CODE REVIEW

File: client/checkout/platform-checkout/email-input-iframe.js [hunk 1]
```diff
@@ -366,8 +368,6 @@ export const handlePlatformCheckoutEmailInput = ( field, api ) => {
 		// Insert the wrapper into the DOM.
 		parentDiv.insertBefore( loginSessionIframeWrapper, null );
 
-		setPopoverPosition();
```
Comment (Mayisha) [inline client/checkout/platform-checkout/email-input-iframe.js:377]: This unnecessary function here is the main cause of the auto scroll. This function sets the position of the OTP modal. It's not needed here.

**Review by gpressutto5 — APPROVED** [review]
I thought the scroll was intentional 😅 This PR fixes the issue.

**Comment by fashuvo** [comment]
**Size Change:** -1 B (0%) 

**Total Size:** 1.63 MB

| Filename | Size | Change |
| :--- | :---: | :---: |
| `release/woocommerce-payments/dist/checkout.js` | 25.9 kB | -1 B (0%) |

<details><summary>ℹ️ <strong>View Unchanged</strong></summary>

| Filename | Size |
| :--- | :---: |
| `release/woocommerce-payments/assets/css/admin.css` | 913 B |
| `release/woocommerce-payments/assets/css/success.css` | 401 B |
| `release/woocommerce-payments/dist/blocks-checkout.css` | 1.77 kB |
| `release/woocommerce-payments/dist/blocks-checkout.js` | 27 kB |
| `release/woocommerce-payments/dist/checkout.css` | 1.85 kB |
| `release/woocommerce-payments/dist/index.css` | 35.8 kB |
| `release/woocommerce-payments/dist/index.js` | 627 kB |
| `release/woocommerce-payments/dist/multi-currency-analytics.js` | 8.22 kB |
| `release/woocommerce-payments/dist/multi-currency-switcher-block.js` | 71 kB |
| `release/woocommerce-payments/dist/multi-currency.css` | 14.7 kB |
| `release/woocommerce-payments/dist/multi-currency.js` | 109 kB |
| `release/woocommerce-payments/dist/onboarding.css` | 1.12 kB |
| `release/woocommerce-payments/dist/onboarding.js` | 38 kB |
| `release/woocommerce-payments/dist/order.js` | 299 B |
| `release/woocommerce-payments/dist/payment-gateways.css` | 1.2 kB |
| `release/woocommerce-payments/dist/payment-gateways.js` | 82.8 kB |
| `release/woocommerce-payments/dist/payment-request.js` | 22.4 kB |
| `release/woocommerce-payments/dist/platform-checkout.css` | 3.73 kB |
| `release/woocommerce-payments/dist/platform-checkout.js` | 76.6 kB |
| `release/woocommerce-payments/dist/settings.css` | 47.2 kB |
| `release/woocommerce-payments/dist/settings.js` | 148 kB |
| `release/woocommerce-payments/dist/subscription-edit-page.js` | 3.66 kB |
| `release/woocommerce-payments/dist/subscription-product-onboarding-modal.css` | 7.18 kB |
| `release/woocommerce-payments/dist/subscription-product-onboarding-modal.js` | 16.5 kB |
| `release/woocommerce-payments/dist/subscription-product-onboarding-toast.js` | 428 B |
| `release/woocommerce-payments/dist/subscriptions-empty-state.css` | 298 B |
| `release/woocommerce-payments/dist/subscriptions-empty-state.js` | 48.8 kB |
| `release/woocommerce-payments/dist/tos.css` | 236 B |
| `release/woocommerce-payments/dist/tos.js` | 22.7 kB |
| `release/woocommerce-payments/dist/upe_checkout.css` | 1.85 kB |
| `release/woocommerce-payments/dist/upe_checkout.js` | 37.2 kB |
| `release/woocommerce-payments/dist/upe-blocks-checkout.css` | 1.77 kB |
| `release/woocommerce-payments/dist/upe-blocks-checkout.js` | 43.4 kB |
| `release/woocommerce-payments/includes/subscriptions/assets/css/plugin-page.css` | 633 B |
| `release/woocommerce-payments/includes/subscriptions/assets/js/plugin-page.js` | 355 B |
| `release/woocommerce-payments/vendor/automattic/jetpack-admin-ui/src/css/jetpack-icon.css` | 224 B |
| `release/woocommerce-payments/vendor/automattic/jetpack-assets/build/i18n-loader.js` | 2.43 kB |
| `release/woocommerce-payments/vendor/automattic/jetpack-assets/src/js/i18n-loader.js` | 1.01 kB |
| `release/woocommerce-payments/vendor/automattic/jetpack-identity-crisis/babel.config.js` | 160 B |
| `release/woocommerce-payments/vendor/automattic/jetpack-identity-crisis/build/index.css` | 2.27 kB |
| `release/woocommerce-payments/vendor/automattic/jetpack-identity-crisis/build/index.js` | 13.3 kB |
| `release/woocommerce-payments/vendor/automattic/jetpack-identity-crisis/build/index.rtl.css` | 2.27 kB |
| `release/woocommerce-payments/vendor/automattic/jetpack-tracking/src/js/tracks-ajax.js` | 789 B |
| `release/woocommerce-payments/vendor/automattic/jetpack-tracking/src/js/tracks-callables.js` | 925 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/css/about.css` | 1.2 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/css/admin-order-statuses.css` | 403 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/css/admin.css` | 3.43 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/css/checkout.css` | 299 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/css/modal.css` | 742 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/css/view-subscription.css` | 572 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/css/wcs-upgrade.css` | 411 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/admin-pointers.js` | 544 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/admin.js` | 8.97 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/jstz.js` | 6.8 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/jstz.min.js` | 3.83 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/meta-boxes-coupon.js` | 544 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/meta-boxes-subscription.js` | 2.35 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/moment.js` | 22.1 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/moment.min.js` | 11.6 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/payment-method-restrictions.js` | 1.29 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/admin/wcs-meta-boxes-order.js` | 502 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/frontend/payment-methods.js` | 355 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/frontend/single-product.js` | 429 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/frontend/view-subscription.js` | 1.38 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/frontend/wcs-cart.js` | 387 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/modal.js` | 1.1 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/assets/js/wcs-upgrade.js` | 1.27 kB |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/build/index.css` | 392 B |
| `release/woocommerce-payments/vendor/woocommerce/subscriptions-core/build/index.js` | 3.06 kB |

</details>



<a href="https://github.com/preactjs/compressed-size-action"><sub>compressed-size-action</sub></a>

**Comment by Mayisha** [comment]
Hi @fashuvo. Before merging this PR when I updated the branch [this bundle size check comment](https://github.com/Automattic/woocommerce-payments/pull/4786#issuecomment-1257530802) was posted. But the comment is posted by your user instead of the github-bot. Is this expected? :thinking:

**Comment by fashuvo** [comment]
> Hi @fashuvo. Before merging this PR when I updated the branch [this bundle size check comment](https://github.com/Automattic/woocommerce-payments/pull/4786#issuecomment-1257530802) was posted. But the comment is posted by your user instead of the github-bot. Is this expected? 🤔

Not actually. I didn't know that comments will be authored by me.

I am checking how we can get the comment authored by the github-bot instead.

**Pull request merged**